var/spack/repos/builtin/packages/py-astroid/package.py

```diff
@@ -22,10 +23,15 @@ class PyAstroid(PythonPackage):
     version('1.4.2', sha256='f9007d651f4b3514ea5812127677a4bb681ff194164290cea358987920f24ee6')
     version('1.4.1', sha256='f1ab3ee6f17f9d30981399a52b56a7a7d2747ba24f0aa504e411ee6205a01fc0')
 
```

Comment:
Can you add a comment saying that dependencies can be found in `astroid/__pkginfo__.py`? Took me longer than I would like to figure out where you found these.